eng/WpfArcadeSdk/tools/Packaging.props

```diff
@@ -25,8 +25,8 @@
   <PropertyGroup>
     <GitHubTransportPackageName>Microsoft.DotNet.Wpf.GitHub</GitHubTransportPackageName>
     <DncEngTransportPackageName>Microsoft.DotNet.Wpf.DncEng</DncEngTransportPackageName>
+    <DncEngTestPackageName>Microsoft.DotNet.Wpf.DncEng.Test</DncEngTestPackageName>
```

Comment:
If you're building this in GitHub, it's not really a DncEng package.  I think we should name it based on where it is built, not where it ends up.

Reply:
Maybe I am misunderstanding the intent of the package is this being built solely in int or from GitHub?

Reply:
The name of the package is being added to the props file - that's all. It will get consumed during build orchestration in dotnet-wpf-int where the package named m.d.w.d.test will be produced.

Reply:
solely in int

Reply:
Got it, my main concern at this point is the infra package being built internally, then flipping to GitHub at some point.  Maybe we can just open the infra as well?

Minor concern though, just give it some thought,